### PR TITLE
Add support for asyncio compatible transport

### PR DESCRIPTION
**Pull request by ticosax**
## To install
```bash
pip install opbeat[asyncio]
```

## Usage

```python
from opbeat.contrib.asyncio import Client
from opbeat.handlers.logging import OpbeatHandler

opbeat_client = Client(
    app_id=app_id,
    organization_id=organization_id,
    secret_token=secret_token,
    transport_class='opbeat.transport.asyncio.AsyncioHTTPTransport')

opbeat_handler = OpbeatHandler(opbeat_client, level='ERROR')
logging.getLogger(__name__).addHandler(opbeat_handler)
```

### CODE REVIEW

**Comment by roncohen** [comment]
This is way cool @ticosax. Thanks! We'll take a look.

**Comment by piquadrat** [comment]
Thanks again @ticosax. I played around a bit with it, works great!

File: Makefile [hunk 1]
```diff
@@ -2,10 +2,12 @@ isort:
 	isort -rc -vb .
 
 test:
-	py.test --isort
+	if [ "$$TRAVIS_PYTHON_VERSION" != "3.5" ]; then \
+	py.test --isort --ignore=tests/asyncio; \
+	else py.test --isort; fi
```
Comment (blueyed) [inline Makefile]: @ticosax 
Could this make use of https://pypi.python.org/pypi/pytest-asyncio instead?

File: setup.cfg [hunk 2]
```diff
@@ -7,6 +7,9 @@ universal=0
 
 [pytest]
 python_files=tests.py test_*.py *_tests.py
+isort_ignore=
+    opbeat/transport/asyncio.py
+    opbeat/contrib/asyncio/client.py
```
Comment (blueyed) [inline setup.cfg]: @ticosax 
Why is that?

**Comment by ticosax** [comment]
Nice !

**Pull request merged**